packages/core/src/utils.ts

```diff
@@ -223,3 +229,17 @@ export type Falsy = undefined | null | false | '' | 0;
 export function isTruthy<Truthy>(value: Truthy | Falsy): value is Truthy {
   return !!value;
 }
+
+export function identity<T>(value: T): T {
+  return value;
+}
+
+export type StdArgs = [string | Uint8Array];
+
+export function stderr(...args: StdArgs) {
+  process.stderr.write(...args);
+}
+
+export function stdout(...args: StdArgs) {
+  process.stdout.write(...args);
+}
```

Comment:
I believe we don't really need these façades. There're used each in one place only and shouldn't be further reused.